docs/helping.md

```diff
@@ -74,8 +94,10 @@ Note that `close_delay` is a number, meaning the time in hours before the ticket
 Using "ac" as described later lets other people know how long until the ticket will close. "ac" meaning **A**uto **C**lose.
 
 If the ticket very highly seems resolved/the user has said they have no issues, use `/closerequest` with `close_delay` set to `1` and the `reason` set to "resolved, ac 1hr"\
-If the ticket seems like it _might_ be resolved, try a `close_delay` of `3-8` (use your judgement) with the `reason` "seemingly resolved, ac {HOUR-COUNT}hr". This gives the user more time to deny the request if the issue isn't actually resolved\
-For inactive tickets (when it's the _user_ not responding) we usually give them a few days (3-5, sometimes longer if we forget) then start pinging them once a day. If they continue to not respond for 2-3 days, we tend to set a close request with `close_delay` set to `24` or `48` (`24` is especially nicer for users who _never_ responded to their ticket), and the `reason` set to "no response, ac {HOUR-COUNT}hr" or "inactive ticket, ac {hour count}hr"
+If the ticket seems like it _might_ be resolved, try a `close_delay` of `3-8` (use your judgement) with the `reason` "seemingly resolved, ac {HOUR-COUNT}hr". 
```

Comment:
I'd keep 8 hour auto close the minimum for resolved ^^

Reply:
do you mean _all_ resolved or seemingly resolved?

Reply:
I'd say all resolved. The ticket author can still manually resolve earlier if they wanna ^^